Packs/RubrikPolaris/Integrations/RubrikPolaris/RubrikPolaris.yml

```diff
@@ -2913,7 +2868,7 @@ script:
     - contextPath: RubrikPolaris.GPSVMRecoverFiles.id
       description: Recover files request ID.
       type: String
-  dockerimage: demisto/rubrik-polaris-sdk-py3:1.0.0.27986
+  dockerimage: demisto/rubrik-polaris-sdk-py3:1.0.1.30997
```

Comment:
```suggestion
  dockerimage: demisto/rubrik-polaris-sdk-py3:1.0.0.31195
```

Reply:
@JasBeilin We also should update the docker image in the release notes. Shall we make a new commit? (asking since you have already approved it)